### PR TITLE
Improved documentation

### PR DESCRIPTION
**Pull request by frenck**
## Proposed Changes

Update documentation on several parts:

- To match our Markdown code guidelines

- Updated copyright years in preparation of the new release

- Updated badges/shields

   - Added [project stage](http://bl.ocks.org/potherca/raw/a2ae67caa3863a299ba0/) badge 
   - Added Scrutinizer badge
   - Added latests version on Packagist badge

## Related Issues

This PR prepares for a next release as requested in #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +31,7 @@ Add the following lines to your `composer.json` file:
 
 ## Developing Coding Standards
 
-Coding standard can be developed in the way [PHP_CodeSniffer] documents [this].
+Coding standard can be developed in the way [PHP_CodeSniffer][codesniffer] documents [this][tutorial].
```
Comment (Potherca) [inline README.md]: Please avoid using "this" for links. Suggestion:

    Coding standard can be developed in the way [PHP_CodeSniffer][codesniffer] documents in their [tutorial page][tutorial].

File: README.md [hunk 2]
```diff
@@ -54,46 +52,38 @@ Requirements:
...
-[contribution guidelines]: https://github.com/dealerdirect/phpcodesniffer-composer-installer/blob/master/CONTRIBUTING.md
-
-## Authors & Contributors
+## Authors & contributors
 
-The original idea and setup of this repository is by [Franck Nijhof], employee @ Dealerdirect.
+The original idea and setup of this repository is by [Franck Nijhof][frenck], employee @ Dealerdirect.
 
-For a full list off all author and/or contributors, please check [this page].
+For a full list off all author and/or contributors, please check [this page][contributors].
```
Comment (Potherca) [inline README.md]: Please avoid using "this" for links. Suggestion:

    For a full list off all author and/or contributors, please check [the contributors page][contributors].

**Comment by Potherca** [comment]
Does exactly what it says on the tin

Changes look good. Minor request for change of existing text (see review comments).

**Comment by frenck** [comment]
@Potherca Updated this PR with the requested changed.

**Pull request merged**